src/Functions/URL/ExtractFirstSignificantSubdomainDefaultLookup.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
```

Comment:
It's an implementation detail of the lookup function, why does it need its own header?

Reply:
Just to make it similar to `src/Functions/URL/ExtractFirstSignificantSubdomainCustomLookup.h`

Reply:
That one doesn't look like it deserves a header either :)

Reply:
Agree.

TL;DR; at first it was included into two files, however after few rebases it goes away.